packages/reporters/build-metrics/src/BuildMetricsReporter.js

```diff
@@ -4,12 +4,25 @@ import path from 'path';
 import {Reporter} from '@parcel/plugin';
 import {generateBundleReport} from '@parcel/utils';
 
+let timings = {};
```

Comment:
Could two concurrent Parcel instances use the same reporter? Maybe use some kind of map?

Reply:
it's mainly used for the benchmark action which runs one after another so that should probably never happen, however someone else might, not entirely sure how I'd give a run a unique id I'll look if there's some kind of hash in the options or something I can use

Reply:
Can't really seem to find any id I could use for this, unless I'd hash options... but that seems kinda a lot to do on every event

Reply:
Maybe we should pass a build id as part of the options? I think @wbinnssmith might have mentioned needing something like that for something else maybe

Reply:
Seems like the only way to me without impacting performance too much